generators/app/templates/src/app/shell/__ionic-side-menu._shell.component.html

```diff
@@ -1,12 +1,14 @@
 <ion-split-pane>
-  <ion-menu [content]="content">
+  <ion-menu>
 <% if (!props.auth) { -%>
     <ion-header>
       <ion-toolbar color="primary">
         <ion-buttons *ngIf="isWeb" end>
-          <button ion-button icon-only (click)="changeLanguage()">
+          <ion-button class="profile-button" color="light" (click)="showProfileActions()"
+            round fill="outline" size="small" slot="start"
+          >
```

Comment:
`>` should be on the previous line to keep it consistent with existing code style

Reply:
moved :)